apps/src/StudioApp.js

```diff
@@ -3039,25 +3077,30 @@ function rectFromElementBoundingBox(element) {
  * @param {string} type - Alert type (error, warning, or notification)
  * @param {React.Component} alertContents
  */
-StudioApp.prototype.displayWorkspaceAlert = function(type, alertContents) {
-  var container = this.displayAlert(
-    '#codeWorkspace',
-    {type: type},
-    alertContents
+StudioApp.prototype.displayWorkspaceAlert = function(
+  type,
+  alertContents,
+  bottom = false
+) {
+  var parent = $(bottom && this.editCode ? '#codeTextbox' : '#codeWorkspace');
+  var container = $('<div/>');
+  parent.append(container);
+  ReactDOM.render(
+    <WorkspaceAlert
```

Comment:
🎉

Reply:
please do continue to avoid using JSX syntax inside non-JSX files

Reply:
@Hamms - what's the preferred way to write this?

Reply:
I just switched it to use createElement like you suggested earlier, let me know if this wasn't what you meant

Reply:
oh, sorry I missed this. Yep, `createElement` is preferred! Thanks